job-tracker/src/App.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import firebase from "./utils/firebase";
+
+// components
+import Header from "./layout/Header";
+import Footer from "./layout/Footer";
+import AddApplication from "./components/AddApplication";
+import EditJob from "./components/EditJob";
+import ApplicationList from "./components/ApplicationList";
+
+require("./styles/main.scss");
```

Comment:
You have to be consistent with your styles, just install `yarn add node-sass` and you will be able to import scss files using the ES6 way
```suggestion
import "./styles/main.scss";
```

Reply:
I do use node-sass.
I got used to this style of writing it after my internship for some reason.

Reply:
I understand that but as your app grows in size and if you need to add more scss files then you will need to use node-sass